core/internal/cltest/factories.go

```diff
@@ -147,7 +147,24 @@ func NewJobWithFluxMonitorInitiator() models.JobSpec {
 		InitiatorParams: models.InitiatorParams{
 			Address:     NewAddress(),
 			RequestData: models.JSON{gjson.Parse(`{"data":{"coin":"ETH","market":"USD"}}`)},
-			Feeds:       []string{"https://lambda.staging.devnet.tools/bnc/call"},
+			Feeds:       models.JSON{gjson.Parse(`["https://lambda.staging.devnet.tools/bnc/call"]`)},
```

Comment:
it does, but `cltest.JSONFromString` requires a `testing` object as the first parameter, so it would instead look like...

```go
Feeds:       cltest.JSONFromString(nil, `["https://lambda.staging.devnet.tools/bnc/call"]`),
```

I'm okay with either - here I was just following what we did with `RequestData`